src/Symfony/Bundle/SecurityBundle/Tests/Functional/AutowiringTypesTest.php

```diff
@@ -22,6 +22,15 @@ public function testAccessDecisionManagerAutowiring()
         $this->assertInstanceOf('Symfony\Component\Security\Core\Authorization\AccessDecisionManager', $accessDecisionManager);
     }
 
+    public function testDebugAccessDecisionManagerAutowiring()
+    {
+        static::bootKernel(array('debug' => true));
+        $container = static::$kernel->getContainer();
+
+        $accessDecisionManager = $container->get('test.autowiring_types.autowired_services')->getAccessDecisionManager();
+        $this->assertInstanceOf('Symfony\Component\Security\Core\Authorization\AccessDecisionManager', $accessDecisionManager);
```

Comment:
yes, this case exist in #19684 (targeted 3.1).

see https://github.com/symfony/symfony/pull/19684#issuecomment-242914681,

> DebugAccessDecisionManager has been introduced in 3.1